### ISSUE TITLE
V2 update?

### ISSUE THREAD

**Issue by ranrinc** [cyberjunky/home-assistant-google_fit#8]
Just wondering since V1 is depreceated and HA is throwing error messages. Will you going to implement V2 API soon?

**Comment by syphernl** [comment]
ISTM that this integration needs to have additional scopes added to be able to read this data:
* https://www.googleapis.com/auth/fitness.sleep.read
* https://www.googleapis.com/auth/fitness.heart_rate.read

I tried to add these to my fork, this seems to be failing because of:
```
Error: Invalid response 400. Error: invalid_scope
You will need to restart hass after fixing.
```

Either way I was able to get it to work again using this: https://community.home-assistant.io/t/google-fit-support/4556/182

With the file it generated I can now see my sleep data + heartrate data again in HomeAssistant _without_ any other code changes needed to this integration.

According to that particular thread it is supposedly not possible to use the `code` authentication with these sensitive scopes.
In another fork of this there is a discussion about how to fix this: https://github.com/vmanuel/hacs-google-fit/issues/13. 
The http 400 is also being mentioned/debugged here: https://github.com/vmanuel/hacs-google-fit/issues/15

**TLDR**: This integration needs some changes for it to work. Using the mentioned HASS Community link you can generate a working `.google_fit.token` file that makes heartrate and sleep tracking work again.

**Comment by sharoltd** [comment]
Hello to all! Did someone solve the problem with HeartRate and Sleep sensors? Is there a working solution?
Thanks!

**Comment by cyberjunky** [comment]
Can you try version 1.0.7?
I added the scopes and it kept working, i have no heart/sleep data to test with.

**Comment by syphernl** [comment]
> Can you try version 1.0.7?
> I added the scopes and it kept working, i have no heart/sleep data to test with.

Everything shows up fine here 👍🏻

**Comment by cyberjunky** [comment]
@syphernl including sleep and heart rate?
I can't get it to work when i create manual sleep sessions

**Comment by syphernl** [comment]
> @syphernl including sleep and heart rate? I can't get it to work when i create manual sleep sessions

Yes, I see both my heartrate and sleep duration as reported to Google Fit via Zepp Life (previously Mi Fit).

**Comment by sharoltd** [comment]
Try to install it again. Have error:
Error: Invalid response 400. Error: invalid_scope
after reload HA. ((
